src/selector_gui.cpp

```diff
@@ -0,0 +1,386 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file selector_gui.cpp A composable selector widget. */
+
+#include "3rdparty/fmt/core.h"
+#include "stdafx.h"
```

Comment:
`stdafx.h` needs to come first
No other source file includes `3rdparty/fmt/core.h` so I'm sceptical that you need to here.